connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -179,10 +181,13 @@ export async function githubRepoIssuesSyncWorkflow({
   });
   const promises: Promise<void>[] = [];
 
+  await githubUpsertIssuesDirectoryActivity({ connectorId, repoId });
+
```

Comment:
note: since the workflow code changed you're gonna have to work your way around Temporal non-deterministic error by stopping the workflows and restarting (very dangerous because of webhooks I think).
I'm gonna look for a solution where only activity code would change first thing tomorrow

Reply:
hmm there isn't really another place to put this folder upsert, let's discuss this tomorrow 👍